javascript/src/main/components/TutorAssignment/TutorAssignmentTable.js

```diff
@@ -13,9 +18,9 @@ export default ({tutorAssignments, isInstructor}) => {
         )
     }
 
-    const renderDeleteButton = (_id) => {
+    const renderDeleteButton = (id) => {      
         return (
-            <Button variant="danger" data-testid="delete-button" >Delete</Button>
+            <Button variant="danger" data-testid="delete-button" onClick={() => deleteTutorAssignment(id)}>Delete</Button>
         )
     }
 
```

Comment:
You may want to put this change in a separate branch.  There is no point in having the UI change until we have the backend working.   Putting the backend into its own branch will simplify things, and you may even earn more points. (You certainly won't earn fewer.)

I would split this into two stories: one for the backend (which you should PR first) and a separate one for the front end (which you should PR second).